helm_deploy/laa-court-data-adaptor/public-values/uat.yaml

```diff
@@ -21,7 +21,7 @@ sqs_region: eu-west-2
 sqs_messaging_enabled: true
 maat_api:
   validation_enabled: true
-  oauth_url: https://maat-cd-api-uat.auth.eu-west-2.amazoncognito.com
+  oauth_url: https://laa-maat-data-api-uat.apps.live.cloud-platform.service.justice.gov.uk
```

Comment:
This looks like the api_url, not the oauth_url

Reply:
My bad, copy paste issue. Fixed it